### PR TITLE
Add issue templates

### LINKED ISSUE #288: Add Agile Templates
# Story

As a Primaza Developer,
I would like to have templates for Issue/Story/Task/Bug/Epic
so that it would be easier to use GitHub Project

## In Scope

* GitHub Templates for Issue/Story/Task/Bug/Epic

## Approach

Clean and adapt to GitHub Project, the already existing templates.

## Demo requirements

Discuss in Community Call the new templates and the User Experience using them.

## Dependencies

NA

## Acceptance Criteria

* Development
:question: Add GitHub Project templates

 * Docs
:question: Updating the `CONTRIBUTING.md` file

### Legend

:question: Unknown
:heavy_check_mark: Verified
:x: Unsatisfied

### PR DESCRIPTION
**Pull request by filariow**
resolves #288

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+**To Reproduce**
+_Steps to reproduce the behavior:_
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+_A clear and concise description of what you expected to happen._
+
```
Comment (sadlerap) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]: We should also have an actual behavior section.  What behavior should we have seen, and what behavior did we actually see?

Comment (filariow) [reply .github/ISSUE_TEMPLATE/bug_report.md:22]: The one I ended up pushing is the one suggested by GitHub.

I had the same concern, but then thought that for how the section is structured the actual behavior is detailed in the steps to reproduce.
Also I thought that a lot of time, I saw the actual behavior section being a repetition of the steps.

I was waiting for a discussion like this one to decide whether or not to have the "actual behavior" section. I'd propose not to for now, and add it in future if we feel we need it.

Based on your experience, what do you think? Would it be enough just having the "steps to reproduce" block?

Comment (sadlerap) [reply .github/ISSUE_TEMPLATE/bug_report.md:22]: With this context, I'm fine with it either way, though I'd like to hear some other opinions.

**Review by sadlerap — APPROVED** [review]
lgtm - if we run into issues with these going forward, we can always change them.

**Pull request merged**